packages/documentation/components/NavBar.vue

```diff
@@ -25,17 +26,16 @@ import { useRouter } from '../hooks/use-router'
 
 export default defineComponent({
 	name: 'NavBar',
-	props: {
-		isNarrow: { type: Boolean, default: false },
-	},
 	setup() {
 		const route = useRoute()
 		const router = useRouter()
+		const isNarrow = ref(false)
```

Comment:
Would be amazing if the docs could persist it to localStorage. I personally always want the navbar to be collapsed as kotti needs a lot of space right now